pkg/sqlite/load.go

```diff
@@ -1465,14 +1535,12 @@ func (s *sqlLoader) DeprecateBundle(path string) error {
 	if err != nil {
 		return err
 	}
-
 	// Clean up the deprecated table by dropping all truncated bundles
 	// (see pkg/sqlite/migrations/013_rm_truncated_deprecations.go for more details)
-	_, err = tx.Exec(`DELETE FROM deprecated WHERE deprecated.operatorbundle_name NOT IN (SELECT DISTINCT deprecated.operatorbundle_name FROM (deprecated INNER JOIN channel_entry ON deprecated.operatorbundle_name = channel_entry.operatorbundle_name))`)
+	_, err = tx.Exec(`DELETE FROM deprecated WHERE deprecated.operatorbundle_name NOT IN (SELECT DISTINCT deprecated.operatorbundle_name FROM (deprecated INNER JOIN operatorbundle ON deprecated.operatorbundle_name = operatorbundle.name))`)
```

Comment:
seems that would also solved : https://bugzilla.redhat.com/show_bug.cgi?id=2001954

Reply:
Solved by https://github.com/operator-framework/operator-registry/pull/780/files